.github/workflows/overcloud-host-image-build.yml

```diff
@@ -390,3 +390,41 @@ jobs:
           OS_APPLICATION_CREDENTIAL_ID: ${{ secrets.OS_APPLICATION_CREDENTIAL_ID }}
           OS_APPLICATION_CREDENTIAL_SECRET: ${{ secrets.OS_APPLICATION_CREDENTIAL_SECRET }}
         if: always()
+
+      - name: Trigger update overcloud host image tags
```

Comment:
Do we want this to trigger every time?

We can keep it on by default, but it'd be nice to have an option to skip the tag update since sometimes we just want to test that images do in fact build with a new configuration

Reply:
I hadn't considered that use case, I'll add a way to skip it

Reply:
Done